pkgs/development/python-modules/pybind11/default.nix

```diff
@@ -23,6 +23,14 @@ buildPythonPackage rec {
     sha256 = "1lsacpawl2gb5qlh0cawj9swsyfbwhzhwiv6553a7lsigdbadqpy";
   };
 
+  patches = [
+    # fix pybind11Config.cmake
+    (fetchpatch {
+      url = "https://github.com/pybind/pybind11/commit/d9c4e1047a95f023633a7260af5a633307438941.patch";
```

Comment:
Is this URL stable? I am not sure when commits are preserved, but I can imagine that if e.g. the PR commit is amended because upstream asks for changes that this particular SHA disappears?

Reply:
I honestly don't know but I've done this quite often in the past. Since I created the PR, I'll make sure to test this if the commit is amended.

Reply:
I think dangling commits are kept around.

Once in master, the nixos cache should retain it as well.

Reply:
Good to know! I know that once Hydra caches it, that it'll be around, but some people build without Hydra.